admin/move_attachments.php

```diff
@@ -257,20 +257,24 @@ function move_attachments_to_disk( $p_type, array $p_projects ) {
 	return $t_moved;
 }
 
-$t_array = explode( ':', $f_project_to_move, 2 );
-if( isset( $t_array[1] ) ) {
-	$f_project_id = $t_array[1];
-
-	if( !is_numeric( $f_project_id ) || (int)$f_project_id == 0 ) {
-		$t_moved = array();
-	} else {
-		switch( $t_array[0] ) {
-			case 'disk':
-				$t_moved = move_attachments_to_disk( $f_file_type, array( $f_project_id ) );
-				break;
-			case 'db':
-				$t_moved = move_attachments_to_db( $f_file_type, array( $f_project_id ) );
-				break;
+$t_moved = array();
+
+if ( null != $f_project_to_move ) {
+	foreach ( $f_project_to_move as $t_project_to_move) {
+		
+		$t_array = explode( ':', $t_project_to_move );
+
+		if( isset( $t_array[1] ) ) {
+			$f_project_id = $t_array[1];
```

Comment:
$f_field is only for form fields and not for temporary / calculated fields.  Please change to $t_project_id.